### PR TITLE
Enhance footer

### PR DESCRIPTION
**Pull request by ahennr**
This PR adds the following components to the footer:

*  `CoordinateReferenceSystemCombo` (containing `EPSG:4326` and `EPSG:3857` by default)
* Mouse position
* Imprint / privacy

Please review @terrestris/devs

### CODE REVIEW

File: packages/baseclient-components/src/container/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+} from 'ol/proj.js';
+import { applyTransform } from 'ol/extent.js';
+import { createStringXY } from 'ol/coordinate.js';
+import OlView from 'ol/View';
+import OlMousePositionControl from 'ol/control/MousePosition';
+
+import {
+  CoordinateReferenceSystemCombo,
+  ScaleCombo
+} from '@terrestris/react-geo/';
```
Comment (annarieger) [inline packages/baseclient-components/src/container/Footer/Footer.tsx]: extra `/` after `react-geo`

File: packages/baseclient-components/src/container/Footer/Footer.tsx [hunk 2]
```diff
@@ -22,13 +39,77 @@ interface FooterState {
...
+    });
+
+    map.addControl(mousePositionControl);
+  }
+
+  /**
+   *
+   * @param crsObj
+   */
+  setProjection(crsObj: any) {
```
Comment (annarieger) [inline packages/baseclient-components/src/container/Footer/Footer.tsx]: Better docs would be nice

File: packages/baseclient-components/src/container/Footer/Footer.tsx [hunk 3]
```diff
@@ -22,13 +39,77 @@ interface FooterState {
...
+   */
+  createOlMousePositionControl = (map: any) => {
+    const mousePositionControl = new OlMousePositionControl({
+      coordinateFormat: createStringXY(2),
+      projection: map.getView().getProjection().getCode(),
+      target: document.getElementById('mouse-position'),
+      undefinedHTML: '&nbsp;'
+    });
+
+    map.addControl(mousePositionControl);
```
Comment (dnlkoch) [inline packages/baseclient-components/src/container/Footer/Footer.tsx]: Please check if the `ol/control/MousePosition` has been added to the map already.

File: packages/baseclient-components/src/container/Footer/Footer.tsx [hunk 4]
```diff
@@ -22,13 +39,77 @@ interface FooterState {
...
+   * @param {Object} nextProps The next props.
+   */
+  componentDidMount() {
+    const {
+      map
+    } = this.props;
+    if (map) {
+      this.createOlMousePositionControl(map);
+    }
+  }
```
Comment (dnlkoch) [inline packages/baseclient-components/src/container/Footer/Footer.tsx]: Even if it's very unlikely, that the Footer will be unmounted, but you might want to remove the `ol/control/MousePosition` from the map in this case.

**Comment by ahennr** [comment]
Thx for reviews @annarieger , @dnlkoch 

I'll merge as soon as travis is happy.

**Pull request merged**